contestants.json

```diff
@@ -79,6 +79,14 @@
             "facebook" : "https://www.facebook.com/joglekaraayush",
             "github" : "https://github.com/aayushjoglekar",
             "twitter" : "https://github.com/aayushjoglekar"
+        },
+        {
+            "name" : "Priyesh Kumar",
+            "imageurl" : "",
```

Comment:
this is a required parameter, please upload an image too.For help take at other participants entry.